### ISSUE TITLE
A11y_code.visualstudio.com_Download_PlatformSpecific_ScreenReader: The success message which appears on download the required built is not announced to the screen reader users.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/vscode-docs#2820]
“[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-manai%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
URL: [https://code.visualstudio.com](https://code.visualstudio.com) 
Windows Version: Windows10
Browser Version: 
Microsoft Edge 44.18362.1.0
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Screen Reader Versions : NVDA2019.1.1
MAS Violated: MAS 4.1.3 (WCAG 4.1.3)

**Repro Steps:**
Navigate to the url: [https://code.visualstudio.com/Download#](https://code.visualstudio.com/Download#)
follow below steps:
1.	Launch the application Url: [https://code.visualstudio.com](https://code.visualstudio.com) 
2.	Navigate to "Download" button.
3.	Start screen reader.
4.	Hit on any of the platform (Windows/Linux/MacOS) links or any installer type (*64 and *32).
5.	Observe the success message and listen whether it is announced

**Actual:**
When screen reader users navigate to platform (Windows/Linux/MacOS) and activate any installer type (*64 and *32) links, the page gets redirected to another page and a success message appears as “Thanks for downloading VS Code for Windows! …..” which is not announced to the screen reader users as soon as it appears on the screen.

**Expected:**
When screen reader users activate any installer type (*64 and *32) links, the page will get redirected to another page and as soon as the success message appears it should be announced to the screen reader users.

**Recommendations:**
This issue can be resolved by providing the success message with the aria-live=”polite” attribute.

Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact:**
The success message is not announced to the screen reader users as soon as it appears on the screen then the users might not know whether the content is downloaded or not.

**MAS Reference:**
MAS 4.1.3 -
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7B684573A7-B089-4131-9B39-0009054125D3%7D&file=MAS%204.1.3%20%E2%80%93%20Status%20Messages.docx&action=default&mobileredirect=true&cid=f1ecca2d-d172-4d08-9d40-6fd75b5d2505](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7B684573A7-B089-4131-9B39-0009054125D3%7D&file=MAS%204.1.3%20%E2%80%93%20Status%20Messages.docx&action=default&mobileredirect=true&cid=f1ecca2d-d172-4d08-9d40-6fd75b5d2505)

**Attachment for Reference:**
[Code.visualstudio.com_Download_PlatformSpecific_ScreenReader Success message is not announced.pptx](https://github.com/microsoft/vscode-docs/files/3329969/Code.visualstudio.com_Download_PlatformSpecific_ScreenReader.Success.message.is.not.announced.pptx)

**Comment by gregvanl** [comment]
Closing for now since the WCAG 2.1 standard is not currently required for Grade C.

**Comment by mstechie** [comment]
GitHubTags:#A11yMAS;
